packages/babel-plugin-flow-to-typescript/src/index.ts

```diff
@@ -81,6 +81,7 @@ export default (_babel: any, opts: PluginOptions = {} as PluginOptions) => {
       parserOpts.plugins.push('objectRestSpread');
       parserOpts.plugins.push('optionalChaining');
       parserOpts.plugins.push('nullishCoalescingOperator');
+      parserOpts.plugins.push('decorators-legacy');
```

Comment:
I am not very familiar with using decorators, what is the difference between this and `decorators` plugin?

as an option this probably can be: 
```js
parserOpts.plugins.push([
    'decorators',
    {
      decoratorsBeforeExport: true,
    },
  ]);
```
but I am not sure what would be actual difference…

Reply:
if, I understand correctly:
- `decorators-legacy` - is stage 1 proposal version
- `decorators` - current, stage 2 proposal version, which looks to be available to use in TypeScript

I guess, we should use a new one…

Reply:
I tried both and both worked but was not sure which one to use. I pushed a commit.